src/selectivity-multiple.js

```diff
@@ -107,10 +107,13 @@ var callSuper = Selectivity.inherits(MultipleSelectivity, {
         'change .selectivity-multiple-input': function() { return false; },
         'click': '_clicked',
         'click .selectivity-multiple-selected-item': '_itemClicked',
+        'mouseover .selectivity-multiple-input-container': 'mouseover',
+        'mouseleave .selectivity-multiple-input-container': 'mouseout',
```

Comment:
Oh shoot, you're right. I have a different project where the events in base classes automatically get merged with those defined in subclasses, but that's not gonna fly here :) So yeah, just leave them here then.

I still think you can simplify it a bit though by omitting the selector, just like the "click" and "change" events. For mouseover you should be covered by the bubbling, and mouseleave gets fired at the higher-up elements too, right?